modules/install-sno-monitoring-the-installation-manually.adoc

```diff
@@ -8,11 +8,15 @@
 
 Use `openshift-install` to monitor the progress of the single-node cluster installation.
 
+.Prerequisites
+
+* Ensure that the boot drive order in the server BIOS settings defaults to booting the server from the target installation disk.
+
 .Procedure
 
 . Attach the modified {op-system} installation ISO to the target host.
```

Comment:
This is the one instance I noticed where ISO is used instead of ISO image.

Reply:
Updated to "Attach the discovery ISO image to the target host."